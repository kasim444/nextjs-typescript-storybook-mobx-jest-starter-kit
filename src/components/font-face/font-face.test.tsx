import * as React from 'react';
import renderer from 'react-test-renderer';

import FontFact from './font-face';

describe('FontFact', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<FontFact />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
