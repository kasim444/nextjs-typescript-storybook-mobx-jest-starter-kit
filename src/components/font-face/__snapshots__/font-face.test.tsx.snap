// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FontFact should match snapshot 1`] = `
<style
  dangerouslySetInnerHTML={
    Object {
      "__html": "@font-face {
  font-family: 'Nexa';
  src: url('/fonts/Nexa_Free_Bold-webfont.woff2') format('woff2'),
    url('/fonts/Nexa_Free_Bold-webfont.woff') format('woff');
  font-weight: normal;
  font-style: normal;
  font-display: swap;
}",
    }
  }
/>
`;
